mne/io/eeglab/eeglab.py

```diff
@@ -717,15 +717,15 @@ def _read_annotations_eeglab(eeg):
         description = []
     elif isinstance(eeg.event, np.ndarray):
         description = [str(event.type) for event in eeg.event]
-        onset = [event.latency for event in eeg.event]
+        onset = [event.latency - 1 for event in eeg.event]
         if (len(onset) > 0) and hasattr(eeg.event[0], 'duration'):
             duration = [event.duration for event in eeg.event]
         else:
             duration = np.zeros(len(onset))
     else:
         # only one event - TypeError: 'mat_struct' object is not iterable
         description = [str(eeg.event.type)]
-        onset = [eeg.event.latency]
+        onset = [eeg.event.latency - 1]
         duration = getattr(eeg.event, 'duration', np.zeros(1))
```

Comment:
as you can see there is something broken here as onset in Annotations should be in seconds but I will fix it in https://github.com/mne-tools/mne-python/pull/4936

Reply:
Ah, right. Ok, so I'll leave it the way it is now here.